script.js

```diff
@@ -81,4 +82,50 @@ function changeStatus(changeId) {
 	} else {
 		btn.setAttribute("value", "作業中");
 	}
-} 
\ No newline at end of file
+	//事後処理
+	switchDisplay();
+} 
+
+
+function switchDisplay() {

```

Comment:
以下のように関数式とアロー関数を用いて関数を定義しましょう！

```js
const hoge = ()=> {
  処理
}
```

Reply:
今回の場合、戻り値を返すことは特にしなくてよいかと思っているのですが、
例として挙げていただいた戻り値変数hogeにする必要性を教えていただけますでしょうか。

Reply:
言葉足らずで申し訳ございません
hogeという名前は例えですので、関数名はご自身のコードに合わせていただいて構いません

>戻り値を返すことは特にしなくてよい

おっしゃる通りです。戻り値は必要ありません。
今回ご指摘させていただいたのは、あくまで関数定義の書き方です！

Reply:
後半の文章「アロー関数にすると、3つ目のコメントにあるようなイベントで、関数名で呼び出しできないと思うのですが、いかがでしょうか」にコメントありませんが、参考に紹介してくださったということですよね？

Reply:
@shouya31 さんのコメントされていた

>以下のように関数式とアロー関数を用いて関数を定義しましょう！

という指摘は以下のようにした方がよいよということを言っていました。
```suggestion
const switchDisplay = () => {
```

>アロー関数にすると、3つ目のコメントにあるようなイベントで、関数名で呼び出しできないと思うのですが、
いかがでしょうか

こちらのコメント内の `3つ目のコメント` がどれに該当するのかが見当たらず…。
お手数ですがどのコメントかを明示いただけますでしょうか？